js/main.js

```diff
@@ -0,0 +1,118 @@
+const pictureCount = 25;
+const avatarMinimalCount = 1;
+const avatarMaximalCount = 6;
+const likeMinimalCount = 15;
+const likeMaximalCount = 200;
+const commentMinimalCount = 1;
+const commentMaximalCount = 10;
+
+const names = [
+  'Лиза',
+  'Ангелина',
+  'Кристина',
+  'Саша',
+  'Женя',
+  'Олег',
+  'Лола',
+  'Даша',
+  'Петя',
+  'Даня',
+  'Ника',
+  'Эвелина',
+  'Лев',
+  'Миша',
+  'Паша',
+  'Сережа',
+  'Коля',
+  'Настя',
+  'Аня',
+  'Надя',
+  'Карина',
+  'Зоя',
+  'Эмили',
+  'Николь',
+  'Милана'
+];
+
+const descriptions = [
+  'Тост с беконом',
+  'Моя собака',
+  'Вид из окна',
+  'Мои подружки',
+  'Кукурузный суп-пюре',
+  'Путешествия 2020',
+  'Токио и Сеул',
+  'Цели 2023',
+  'Маникюр',
+  'Розовый - хит сезона',
+  'Жизнь без глютена',
+  'Жизнь без глюкозы',
+  'Жизнь без удовольствий',
+];
+
+const messages = ['Всё отлично!',
+  'В целом всё неплохо. Но не всё.',
+  'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.',
+  'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.',
+  'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
+  'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!'];
+
+
+const getRandomPositiveInteger = function (a, b) {
+  const lower = Math.ceil(Math.min(Math.abs(a), Math.abs(b)));
+  const upper = Math.floor(Math.max(Math.abs(a), Math.abs(b)));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
+};
+
+/*const getRandomValue = function (min, max) {
```

Comment:
Давай попробуем использовать эту функцию, но только чуть упростим её

Она должна выглядеть примерно так

```suggestion
const createRandomIdGenerator = (min, max) => {
    const ids = // делаем массив с числами от min до max (например при помощи Array.from)
    
    return () => {
        const randomIndex = // получаем рандомных индекс для массива ids (Функция getRandomPositiveInteger здесть отлично подойдет)
        const randomId = // теперь по индексу randomIndex нам нужно достать значение из массива ids и удалить его от туда, чтобы он был уникальным и не повторялся (здесь отлично подойдет метод Array.splice)
        
        // В этом месте может произойти ситуация, когда у нас не осталось элементов в массиве ids и значением переменной randomId будет undefined. В идеале здесь можно добавить if с проверкой и, например, выбрасываением ошибки. Но давай пока что сделаем без этого)
        
        return randomId; // возвращаем значение. Конец :)
    }
}
```